packages/components/src/Actions/Action/Action.component.js

```diff
@@ -90,14 +90,29 @@ function Action(props) {
 
 	const buttonProps = getPropsFrom(Button, rest);
 	const style = link ? 'link' : bsStyle;
-	const rClick = onClick && (event => onClick(event, {
-		action: { label, ...rest },
-		model,
-	}));
+	const rClick = onClick && ((event) => {
+		// to prevent odd bug with FF
```

Comment:
Yes to better understand what happen on Chrome and FF see here https://jsfiddle.net/5zbvw0g8/10/
In the console, when you you middle-click : 
- FF will trigger mouseDown AND mouseClick
- Chrome will trigger only mouseClick
Strangely, it behaves the same with native JS https://jsfiddle.net/sylvainmaes/pfmnz1tf/